T/TopOpt_in_PETSc/bundled/Makefile

```diff
@@ -0,0 +1,20 @@
+PETSC_DIR = $(libdir)/petsc/double_real_Int32
+PETSC_ARCH = x86_64-linux-gnu_double_real_Int32
```

Comment:
What's this used for?

Reply:
It's a relic from using the PETSc build system. I'll remove it shortly, then we can merge! Thanks :)